src/librustdoc/clean/mod.rs

```diff
@@ -2157,12 +2157,26 @@ impl Clean<Vec<Item>> for doctree::Import<'_> {
             return Vec::new();
         }
 
+        let inlined = self.attrs.lists(sym::doc).has_word(sym::inline);
+        let pub_underscore = self.vis.node.is_pub() && self.name == kw::Underscore;
+
+        if pub_underscore && inlined {
+          rustc_errors::struct_span_err!(
+            cx.tcx.sess,
+            self.attrs.lists(sym::doc).next().unwrap().span(),
```

Comment:
This might be wrong if there are multiple `doc` attributes. Can you add a test case for
```rust
#[doc = "xxx"]
#[doc(inline)]
pub use foo::Foo as _;
```

Reply:
This was working as is, maybe because of [https://github.com/rust-lang/rust/blob/master/compiler/rustc_ast/src/ast.rs#L496](https://github.com/rust-lang/rust/blob/master/compiler/rustc_ast/src/ast.rs#L496).
Just to be safe it was changed and made some other refactoring to use the previous calculated values.